tests/testsuite/test.rs

```diff
@@ -4792,6 +4792,35 @@ error: test failed, to rerun pass `--test t1`
 [RUNNING] tests/t2.rs (target/debug/deps/t2[..])
 error: test failed, to rerun pass `--test t2`
 
+Caused by:
+  process didn't exit successfully: `[ROOT]/foo/target/debug/deps/t2[..]` (exit [..]: 4)
+note: test exited abnormally; to see the full output pass --nocapture to the harness.
+",
+        )
+        .with_status(4)
+        .run();
+
+    p.cargo("test --test t2 -- --nocapture")
```

Comment:
Looks like there was a merge problem and the second commit re-adds the test cases the first commit added.

Reply:
Let me take a look at all these commits once, they've gotten a bit confusing. Will push with correct changes in a bit.